wandb/integration/tensorboard/log.py

```diff
@@ -320,22 +330,24 @@ def _log(
             or timestamp - STEPS["global"]["last_log"] >= RATE_LIMIT_SECONDS
         ):
             if history is None:
-                wandb.run._log({})
+                if wandb.run is not None:
+                    wandb.run._log({})
             else:
-                history.add({}, **kwargs)
+                history.add({})
 
         STEPS["global"]["last_log"] = timestamp
 
     if history is None:
-        wandb.run._log(log_dict, commit=False)
+        if wandb.run is not None:
```

Comment:
see previous comment, maybe in this function we could just raise an assertion error (if history is `None`)?

Reply:
same as above